app/Http/Controllers/Api/CommentController.php

```diff
@@ -19,6 +19,7 @@
 use Suyabay\Http\Transformers\CommentTransformer;
 use Suyabay\Http\Transformers\EpisodeTransformer;
 use Suyabay\Http\Transformers\UserCommentTransformer;
+use League\Fractal\Pagination\IlluminatePaginatorAdapter;
```

Comment:
**Code Quality - Nitpick**
@andela-araimi The meta data returned by `IlluminatePaginatorAdapter` gives us a next page link with missing query parameter. Check out how to fix this issue for fractal or make use of laravel's `LengthAwarePaginator `.